perf/index.js

```diff
@@ -13,5 +14,6 @@ test('Large item', function (t) {
     var end = new Date()
     totalDuration += end - start
   }
-  t.ok(totalDuration < 10, 'wanted ' + totalDuration + 'ms to be less than 10ms')
+  var maxTime = iterations * 0.05 // 0.05ms per result
```

Comment:
This is slightly concerning. If the precision is set to 1 then we should be very close to the original value if not the same. Are we missing something?